SEOImages: User Story 1463880, Part 20 (#11346)
dotnet/docs: docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md

> [!NOTE]
>  XML literals and XML axis properties are not supported in declarative code in an ASP.NET page. To use Visual Basic XML features, put your code in a code-behind page in your ASP.NET application.  
  

>  ![Icon that links to a video.](./media/overview-of-linq-to-xml/play-video-icon-example.gif) For related video demonstrations, see [How Do I Get Started with LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-get-started-with-linq-to-xml) and [How Do I Create Excel Spreadsheets using LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-create-excel-spreadsheets-using-linq-to-xml).  

Reviewer: ```suggestion
 ![Play button](./media/overview-of-linq-to-xml/play-video-icon.gif) For related video demonstrations, see [How Do I Get Started with LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-get-started-with-linq-to-xml) and [How Do I Create Excel Spreadsheets using LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-create-excel-spreadsheets-using-linq-to-xml).  
```

Author: @mairaw I've made the changes here. the file name is less than 20 characters and throws a CATS error. Do you have a preference for any additional characters/words?

Reviewer: Argh, sorry about that. Then the original name was good: play-video-icon-example.gif
